python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_decorator.py

```diff
@@ -561,3 +567,76 @@ def downstream_of_dbt():
     assert len(downstream_of_dbt.input_names) == 2
     assert downstream_of_dbt.op.ins["orders"].dagster_type.is_nothing
     assert downstream_of_dbt.op.ins["customized_staging_payments"].dagster_type.is_nothing
+
+
+def test_dbt_with_python_interleaving() -> None:
+    dbt_project_dir = (
+        Path(__file__)
+        .joinpath("..", "dbt_projects", "test_dagster_dbt_python_interleaving")
+        .resolve()
+    )
+    manifest_path = dbt_project_dir.joinpath("manifest.json").resolve()
+    manifest = json.loads(manifest_path.read_bytes())
+
+    # don't select the seeds
+    @dbt_assets(manifest=manifest)
+    def my_dbt_assets():
+        ...
+
+    assert set(my_dbt_assets.keys_by_input_name.values()) == {
+        AssetKey(["dagster", "python_augmented_customers"]),
```

Comment:
Was surprised to see that this is possible without explicitly defining the source in a `.yml` file

Reply:
it does not, just forgot to add the file 🤦